sequel/Gemfile

```diff
@@ -0,0 +1,44 @@
+source 'https://rubygems.org'
+
+module ::Kernel
+  def rails_master?
+    ENV["RAILS_MASTER"] == '1'
```

Comment:
We don't need Rails master here. We should fix Rails to the latest stable version. Sequel should be the only variable here.